src/compiler/types.ts

```diff
@@ -775,8 +775,8 @@ namespace ts {
         expression: Expression;
     }
 
-    /* @internal */
     // A name that supports late-binding (used in checker)
+    /* @internal */

```

Comment:
On second thought, changing the logic such that the `@internal` annotation must be immediately preceding the node isn't a change we probably want to make lightly; I'd keep the old behavior if you're not looking for an inline annotation.